spec/API_specification/array_object.md

```diff
@@ -377,6 +377,23 @@ Evaluates `x1_i & x2_i` for each element `x1_i` of an array instance `x1` with t
 Element-wise results must equal the results returned by the equivalent element-wise function [`bitwise_and(x1, x2)`](elementwise_functions.md#logical_andx1-x2-).
 ```
 
+(method-__bool__)=
+### \_\_int\_\_(x, /)
+
+Converts a dimension-0 boolean array into a Python `bool` object.
```

Comment:
```suggestion
Converts a zero-dimensional boolean array to a Python `bool` object.
```

The general language across the spec has been to use "zero-dimensional" to refer to rank `0` arrays.